meson.build

```diff
@@ -43,6 +43,8 @@ config_h.set('UNUSED', '__attribute__ ((unused))')
 configure_file(output: 'config.h',
   configuration: config_h)
 
+install_man('man/gcolor3.1')
```

Comment:
Can you please move the man page to the data folder? No need for a separate folder.

Reply:
When you do, please move the `install_man` line to `data/meson.build` as well.